src/realm/array_direct.hpp

```diff
@@ -710,11 +593,9 @@ size_t parallel_subword_find(VectorCompare vector_compare, const uint64_t* data,
     while (total_bit_count_left >= fast_scan_limit) {
         // unrolling 2x
         const auto word0 = it.get_with_unsafe_prefetch(bit_count_pr_iteration);
-        it.bump(bit_count_pr_iteration);
         const auto word1 = it.get_with_unsafe_prefetch(bit_count_pr_iteration);
```

Comment:
I think renaming the method will make it clear that it has been moved by  `bit_count_pr_iterations` bits.